app/components/super_selector_component/super_selector_component.html.haml

```diff
@@ -11,11 +11,11 @@
               = item[:label]
         - else
           .super-selector-selected-item
-            = "#{@selected_items.length} categories selected"
+            = t(@selected_items_i18n_key, count: @selected_items.length)
       .selector-arrow{data: reflex_data_attributes(:toggle)}
     .selector-wrapper
       .super-selector-search
-        %input{type: "text", placeholder: "Search", data: reflex_data_attributes("debounced:input->search"), value: @query}
+        %input{type: "text", placeholder: t("components.super_selector.search_placeholder"), data: reflex_data_attributes("debounced:input->search"), value: @query}
```

Comment:
In views you can omit the scope and start the translation key with `.` but I guess that you would have done that if it worked for components. I wonder if that is something that will be added at some point or if we could extend I18n in that way.

Reply:
I didn't know about that scope. I will let as it for now, but maybe I'll change in a near future.